docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -39,6 +39,10 @@ The text element enables you to easily add text to the canvas. The element also
 
 {{< video-embed src="/static/img/docs/canvas-panel/canvas-text-9-2-0.mp4" max-width="750px" caption="Text element demo" >}}
 
+### Ellipse
+
+The ellipse element enables you to add a basic ellipse to the canvas. Ellipse elements support displaying text (both fixed and field data) as well as can change background color based on data thresholds.
```

Comment:
Do you mean the background color of the ellipse element? I've edited assuming that meaning.
```suggestion
The ellipse element lets you add a basic ellipse to the canvas. An ellipse element can display text (both fixed and field data) and its background color can be changed based on data thresholds.
```